dist/d3-timelines.js

```diff
@@ -492,18 +491,16 @@
 						var xpos = -d3Selection.event.transform.x;
 						scroll(xpos, xScale);
 					};
-				};
-
-				var zoom = d3Zoom.zoom()
-					.scaleExtent([0, maxZoom]) // max zoom defaults to 5
-					.translateExtent([[0, 0], [width, 0]]) // [x0, y0], [x1, y1] don't allow translating y-axis
-					.on("zoom", move);
-
-				gParent
-					.classed("scrollable", true)
-					.call(zoom);
-
-				if (! allowZoom) {
+				}
+				if (allowZoom) {
```

Comment:
There needs to be a default zoom.  The default zoom is what allows the sideways scrolling in the `days_scrollable.html`.  Why was this change necessary?

Reply:
Oh, I commented on that jon the PR thread :)

I had a problem with allowZoom. As written, it only allows zoom is allowZoom(false) is configured.
As I understand it should the the oposite, I have change it. If I am wrong, sorry about that, I guess I don't see the semantics/meaning of the allowZoom method :(